src/main/java/com/amazonaws/kinesisvideo/internal/producer/jni/NativeKinesisVideoProducerStream.java

```diff
@@ -529,8 +533,16 @@ public void awaitStopped() throws ProducerException
     {
         // Block until client is stopped or it times out.
         try {
-            if (!mStoppedLatch.await(STOPPED_TIMEOUT_IN_MILLISECONDS, TimeUnit.MILLISECONDS)) {
-                throw new ProducerException("KinesisVideo producer stream stopping time out", ProducerException.STATUS_OPERATION_TIMED_OUT);
+            if(mDeviceInfo.getClientInfo().getStopStreamTimeout() == 0) {
```

Comment:
spacing after if

Reply:
The logic should be 

long timeout = mDevideInfo.getClientInfo().getStopStreamTimeout() / Time.HUNDREDS_OF_NANOS_IN_A_MILLISECOND;

if (timeout == 0) {
timeout = STOPPED_TIMEOUT_IN_MILLISECONDS;
}

 if (!mStoppedLatch.await(STOPPED_TIMEOUT_IN_MILLISECONDS, TimeUnit.MILLISECONDS .....

Reply:
Done

Reply:
Done